app-release-procedures.rst

```diff
@@ -138,113 +119,88 @@ Rollback changes on web worker is relatively simple:
+Same can be achieved via *Heroku UI*.
 
-  1. Before deploying create a rollback database:
+DB migrations can be reverted via `alembic downgrade <http://alembic.zzzcomputing.com/en/latest/tutorial.html#downgrading>`_,
+but use this option only for simple cases. For complex and long running
+migrations consider a creation of rollback database before deployment:
 
   .. code-block::
 
-        sourceheroku addons:create heroku-postgresql:standard-0 --rollback HEROKU_POSTGRESQL_YELLOW --to '2013-10-21 15:52+00' --app sushi
+        $ heroku addons:create heroku-postgresql:standard-0 --rollback HEROKU_POSTGRESQL_YELLOW --to '2013-10-21 15:52+00' --app sushi
 
-  2. Promote the rollback as the primary database:
+and promotion it to the primary DB as a revert step:
 
   .. code-block::
 
         heroku pg:promote HEROKU_POSTGRESQL_SILVER --app sushi
 
-For more detail visit heroku devcenter:
+For more details visit heroku devcenter:
 
     * https://devcenter.heroku.com/articles/heroku-postgres-rollback
     * https://devcenter.heroku.com/articles/releases#rollback
-
-TODO: add some hints on how to rollback for other services that we use
-
-
-Readings
-========
-
-    * https://www.atlassian.com/agile/code-reviews
-    * http://www.plutora.com/blog/release-management-best-practices
-    * https://www.python.org/dev/peps/pep-0350/#mnemonics
     * https://blog.heroku.com/releases-and-rollbacks
-    * https://www.red-gate.com/simple-talk/dotnet/net-framework/the-zen-of-code-reviews-best-practices/
 
 
-**********
+*********
 Templates
-**********
+*********
 
 Basic template:
 ===============
 
-.. code-block:: rst
-
-    Solution
-    --------
+.. code-block:: Markdown
 
+    ## Solution
     **Original issue and supporting materials**:
     **Downsides**:
 
-    Roll-out plan
-    -------------
-
+    ## Roll-out plan
     **Deployment risks**:
     **Deployment assistants**:
     **Backing services related changes**:
     **Backuping steps**:
     **Deployment steps**:
       1. [ ] XXX
 
-    Verification plan
-    -----------------
+    ## Verification plan
 
-    Rollback strategy
-    -----------------
+    ## Rollback strategy
 
 
 Example 1, simple:
 ===================
 
-.. code-block:: rst
-
-    Solution
-    ---------
+.. code-block:: Markdown
 
+    ## Solution
     **Original issue and supporting materials**: #3 increase navbar width
-    **Downsides**: may impact readability on devices with small resolution
-
-    Roll-out plan
-    -------------
+    **Downsides**: solution may impact readability on devices with small resolution
 
+    ## Roll-out plan
     **Deployment risks**: None
     **Deployment assistants**: None
     **Backing services related changes**: None
     **Backuping steps**: None
-
     **Deployment steps**:
-
       1. [ ] Merge branch into master
       2. [ ] Click 'deploy' button on heroku web ui
       3. [ ] Purge cache
       4. [ ] Verify deployment
```

Comment:
Again Heroku does the verify step before the app is released. Here I want to verify that app loads in the browser, monitor paper-trail and getsentry.